target/linux/bcm53xx/base-files/etc/board.d/01_leds

```diff
@@ -4,6 +4,18 @@
 board_config_update
 
 case "$(board_name)" in
+meraki,mx64|\
+meraki,mx64-a0)
+	ucidef_set_led_netdev "wan" "WAN Port" "green:activity-9" "wan"
+	ucidef_set_led_netdev "lan1" "LAN Port 1" "green:activity-1" "lan1"
+	ucidef_set_led_netdev "lan2" "LAN Port 2" "green:activity-3" "lan2"
+	ucidef_set_led_netdev "lan3" "LAN Port 3" "green:activity-5" "lan3"
+	ucidef_set_led_netdev "lan4" "LAN Port 4" "green:activity-7" "lan4"
+	;;
+meraki,mx65)
+	ucidef_set_led_netdev "wan1" "WAN Port 1" "green:activity-1" "wan1"
+	ucidef_set_led_netdev "wan2" "WAN Port 2" "green:activity-3" "wan2"
```

Comment:
Why are these LEDs named "activity" when they are used for ethernet?

Reply:
I have used the rather generic LED_FUNCTION_ACTIVITY, perhaps LED_FUNCTION_WAN/LAN would be appropriate. These labels are generated based on colour, function, enumerator in the DT. Once consensus has been established I can change this upstream.

Reply:
Well, what does the vendor do? OpenWrt follows the vendor setup where possible.
Please do not invent your own LED assignment. Thus, I'd expect the label to correspond to the purpose the vendor implied for that LED.

Reply:
The question is whether LED_FUNCTION_ACTIVITY or LED_FUNCTION_WAN/LAN should be used for port LEDs that flash with network activity as is the case here (and as done by the vendor)? I did consider the "colour:function" scheme in this case. Would the WAN/LAN designation be more appropriate for LEDs that are solid on/off depending on whether or not the WAN/LAN is connected, regardless of network activity?

Reply:
And that's exactly one of the fundamental flaws in that system: How would it be helpful to describe a LED that is (logically) linked to an ethernet port _just_ by its color and the term "activity"? Won't that hide the most important part of the information?